client/my-sites/customer-home/cards/features/reader/index.tsx

```diff
@@ -44,7 +40,7 @@ const ReaderCard = ( props: ReaderCardProps ) => {
 						) }
 					</span>
 				</div>
-				<DiscoverNavigation width={ props.width } selectedTab={ selectedTab } />
+				<DiscoverNavigation selectedTab={ selectedTab } />
```

Comment:
I confirmed this `ReaderCard` is not being used, but it's also wired in pretty tightly with the cards system, so I've just updated it instead of removing it. I even tested enabling it on localhost, and it works with this change. (Though it could use some updates to be shippable.)